<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="param_path" default="$(find-pkg-share autoware_simpl_python)/config/simpl_node.param.yaml"/>
  <arg name="data_path" default="$(find-pkg-share autoware_simpl_python)/data"/>

  <arg name="input/objects" default="/perception/object_recognition/tracking/objects"/>
  <arg name="output/objects" default="objects"/>

  <arg name="lanelet_file" default="lanelet2_map.osm"/>
  <arg name="build_only" default="false"/>

  <node pkg="autoware_simpl_python" exec="autoware_simpl_python_node" name="simpl_python_node" output="screen">
    <param from="$(var param_path)" allow_substs="true"/>
    <remap from="~/input/objects" to="$(var input/objects)"/>
    <remap from="~/output/objects" to="$(var output/objects)"/>
    <param name="lanelet_file" value="$(var lanelet_file)"/>
    <param name="build_only" value="$(var build_only)"/>
  </node>
</launch>
